committeeoversightapp/management/commands/import_data.py

```diff
@@ -0,0 +1,213 @@
+import os
+import csv
+
+from datetime import datetime
+from django.core.management.base import BaseCommand, CommandError
+from django.core.exceptions import ObjectDoesNotExist
+from django.conf import settings
+from django.db import IntegrityError
+from django.db.models import Q
+
+from opencivicdata.core.models import Organization, OrganizationName
+from opencivicdata.legislative.models import Event, EventSource, EventParticipant
+from committeeoversightapp.models import HearingCategory, Committee
+
+jurisdiction_id = 'ocd-jurisdiction/country:us/legislature'
+bad_rows = []
+
+def get_committee(committee_name, parent):
+    classification = "committee"
+    organization = Organization.objects.filter(Q(name__icontains=committee_name, parent=parent, classification=classification) |
+                Q(other_names__name__icontains=committee_name, parent=parent, classification=classification))
+    if organization.count() == 1:
+        return organization[0]
+    elif organization.count() == 0:
+        return IndexError
+    else:
+        return ValueError
```

Comment:
It would be nice to find a way to refactor this, so that you do not need to catch these errors, but instead (1) create the committee and (2) log to `bad_rows`. Right now, it's little abstruse what's going on. Not necessary, of course (since what you have works, and we're running short on hours), but perhaps a meaningful exercise.